components/measure-upload/DateSelectors.tsx

```diff
@@ -1,23 +1,33 @@
 import { Grid } from '@mantine/core';
-import { DateRangePicker } from '@mantine/dates';
+import { DateInput } from '@mantine/dates';
 import { IconCalendar } from '@tabler/icons';
 import { useRecoilState } from 'recoil';
-import { measurementPeriodState } from '../../state/atoms/measurementPeriod';
+import { measurementPeriodEndState, measurementPeriodStartState } from '../../state/atoms/measurementPeriod';
 
 export default function DateSelectors() {
-  const [period, setPeriod] = useRecoilState(measurementPeriodState);
+  const [periodStart, setPeriodStart] = useRecoilState(measurementPeriodStartState);
+  const [periodEnd, setPeriodEnd] = useRecoilState(measurementPeriodEndState);
+
   return (
     <Grid>
-      <Grid.Col span={12}>
-        <DateRangePicker
-          placeholder="Select Measurement Period Range"
-          label="Measurement Period Range"
-          value={[period.start, period.end]}
-          onChange={period => {
-            setPeriod({ start: period[0], end: period[1] });
-          }}
+      <Grid.Col span={6}>
```

Comment:
Because these are now separate date input components, do we want to enforce a constraint that "Start" date comes before "End" date?

Reply:
I did a version of this in https://github.com/projecttacoma/fqm-testify/pull/70/commits/455fb8ecaddefcffc3551fb55908df69b28c01f6

Could probably be improved upon later if there are more kinds of error messages to add